### DISCUSSION TITLE
Fox 24 (WGXA-TV) News

### DISCUSSION THREAD

**Discussion by fjrider** [iptv-org/community#771] [Q&A]
### Channel Name

24 WGXA News

### Channel ID

WGXADT1.us

### Website

https://wgxa.tv/watch

### Notes

Local Macon, Georgia (USA) news channel.

Fox affiliate 24 WGXA News:
https://wgxa.tv/watch

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by Central302** [comment]
[Note] - Sinclair Owned Station 
https://content.uplynk.com/98abe2531c1a4c69920b8a8b683956a6.m3u8?tc=1&exp=1648640445&rn=228965&ct=a&cid=98abe2531c1a4c69920b8a8b683956a6&sig=e198c3220afd071b155eb8008bb9e87bab9d0f33f35a99fa10a1c54743256fac

But... Sadly Tokened. (Not Work Even Remove Paramater)

**Comment by BellezaEmporium** [comment]
https://content.uplynk.com/98abe2531c1a4c69920b8a8b683956a6.m3u8 this should work no ?

**Comment by BellezaEmporium** [comment]
> [Note] - Sinclair Owned Station https://content.uplynk.com/98abe2531c1a4c69920b8a8b683956a6.m3u8?tc=1&exp=1648640445&rn=228965&ct=a&cid=98abe2531c1a4c69920b8a8b683956a6&sig=e198c3220afd071b155eb8008bb9e87bab9d0f33f35a99fa10a1c54743256fac
> 
> But... Sadly Tokened. (Not Work Even Remove Paramater)

Token expired 33140 seconds ago.

**Comment by freearhey** [comment]
https://content.uplynk.com/98abe2531c1a4c69920b8a8b683956a6.m3u8 (offline)
<img width="522" alt="image" src="https://user-images.githubusercontent.com/7253922/232226830-0995923b-2b0d-48cd-a2ec-7aad4df7b606.png">

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19757
